core/register_core_types.cpp

```diff
@@ -96,6 +96,7 @@ static Ref<ResourceFormatLoaderJSON> resource_loader_json;
 static core_bind::ResourceLoader *_resource_loader = nullptr;
 static core_bind::ResourceSaver *_resource_saver = nullptr;
 static core_bind::OS *_os = nullptr;
+static Time *_time = nullptr;
```

Comment:
I would prefer this to be in a different order, so we can keep all the `core_bind` lines together. You could perhaps put this right below `static IP *ip` (which might need to be `_ip`?).